uisdk/src/main/java/com/karhoo/uisdk/screen/booking/domain/quotes/KarhooAvailability.kt

```diff
@@ -76,7 +78,8 @@ class KarhooAvailability(private val quotesService: QuotesService, private val a
                             .quotes(QuotesSearch(
                                     origin = bookingStatusPickup,
                                     destination = bookingStatusDestination,
-                                    dateScheduled = bookingStatus.date?.toDate()))
+                                    dateScheduled = bookingStatus.date?.toDate()),
+                                    locale.toString().replace('_', '-'))
```

Comment:
This can cause `null pointer exception` as the locale is optional and can contain `null` value.